app/models/grda_warehouse/client_file.rb

```diff
@@ -128,7 +133,14 @@ def active_consent_form?
     end
 
     def consent_type
-      if GrdaWarehouse::AvailableFileTag.full_release?(tag_list)
+      if GrdaWarehouse::AvailableFileTag.coc_level_release?(tag_list)
```

Comment:
`coc_level_release?` and `full_release?` are the same, so this will always mask the next branch?

Reply:
So, the overlap is ok.  They two (full_release and coc_level_release) are mutually exclusive.  The translation of the name of the full release string is what will differentiate them.

Reply:
As usual, I was wrong...

Reply:
Maybe this is a fix? https://github.com/greenriver/hmis-warehouse/pull/556/commits/caeaa993b1006aebd66e2e8b523bfd95e1626b14